core/archive/src/main/kotlin/mihon/core/archive/TsZipWriter.kt

```diff
@@ -0,0 +1,33 @@
+package mihon.core.archive
+
+import android.content.Context
+import com.hippo.unifile.UniFile
+import java.io.Closeable
+
+class TsZipWriter(val context: Context, file: UniFile) : Closeable {
```

Comment:
Does this sound better?
```suggestion
class SynchronizedZipWriter(val context: Context, file: UniFile) : Closeable {
```

Reply:
done